backend/endpoints/jokes/blueprint.py

```diff
@@ -1,11 +1,71 @@
-from flask import Blueprint, make_response
+from flask import Blueprint, make_response, request
+from utils.eeljokes import EelJokes, db
+import json
 
 bp = Blueprint(
     "eelbot_joke",
     __name__,
     url_prefix="/joke",
 )
 
-@bp.route("/")
-def main():
-    return make_response(("Hello Eels!", 200))
\ No newline at end of file
+@bp.route("/", methods=['POST'])
+def create_joke():
+    json_data = request.json
+    new_task = EelJokes(JokeText=json_data[0], JokeTextLine2=json_data[1])
+
+    try:
+        db.session.add(new_task)
+        db.session.commit()
+        return "Joke added successfully"
+    except:
+        return "There was an issue adding your joke"
+
+@bp.route("/", methods=['GET'])
+def get_jokes():
+    num_jokes_per_page = int(request.args.get('num_jokes_per_page'))
+    page = int(request.args.get('page'))
+
+    offset = (page - 1) * num_jokes_per_page
+    tasks = EelJokes.query.order_by(EelJokes.JokeID).limit(num_jokes_per_page).offset(offset)
+    return_jokes = []
+    for task in tasks:
+        return_jokes.append(task.get_joke_data())
+    return json.dumps(return_jokes)
+
+@bp.route("/page/", methods=['GET'])
+def pagenum():
+    num_jokes_per_page = int(request.args.get('num_jokes_per_page'))
+
+    import math
+    rows = EelJokes.query.count()
+    num_pages = math.ceil(rows/num_jokes_per_page)
+    return str(num_pages)
```

Comment:
```suggestion
    return  str(math.ceil(rows/num_jokes_per_page))
```

nit: if you aren't doing anything further and just returning the object do so inline.

Reply:
If it was something simple, I would agree with you but there's a few things going on here and having too many things happening in a single line makes it harder to read. I think the original was fine as is.